src/export/CodeSystemExporter.ts

```diff
@@ -28,24 +28,25 @@ export class CodeSystemExporter {
     }
   }
 
-  exportCodeSystem(fshDefinition: FshCodeSystem): void {
-    if (this.codeSystems.some(cs => cs.name === fshDefinition.name)) {
+  exportCodeSystem(fshDefinition: FshCodeSystem): CodeSystem {
+    if (this.pkg.codeSystems.some(cs => cs.name === fshDefinition.name)) {
       return;
     }
     const codeSystem = new CodeSystem();
     this.setMetadata(codeSystem, fshDefinition);
     this.setConcepts(codeSystem, fshDefinition);
-    this.codeSystems.push(codeSystem);
+    this.pkg.codeSystems.push(codeSystem);
+    return codeSystem;
```

Comment:
Yeah.  I was torn on this one.  In the end, I decided they should be consistent.  And it was easier to add the return statement to these than to remove it from the SDExporter since there were already a bunch of tests that relied on that capability of the SDExporter.

TBH -- I'm still not 100% happy w/ how the exporters all work... but I think we're closer.  Still... they may get some more refactoring at some point.

Reply:
Okay! I just wanted to be sure. Yeah I agree it's definitely not worth changing all the tests for SDExporter, so I'm good with this change.